en/components/action-strip.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Action Strip
+_description: The Action Strip represents a template area for one or more actions. The Action Strip requires to be inside a relative container, as it is going to overlay it.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular ActionStrip components, Angular ActionStrip directives, Angular ActionStrip controls
+---
+
+## Action Strip
+<p class="highlight">
+The Ignite UI for Angular [`igxActionStrip`]({environment:angularApiUrl}/classes/igxactionstripcomponent.html) represents a template area for one or more actions. The Action Strip is intended to provide additional UI and functionality for a specific element container. This container should be relative, because the Action Strip attempts to overlay it. This provides an easy way to position the actions over the parent. Even though the parent is overlapped by the Action Strip the main interactions and user access to the container will remain available.
+</p>
+
+<div class="divider--half"></div>
+
+#### Demo
+
+<div class="sample-container loading" style="height: 400px;">
+    <iframe id="action-strip-paragraph-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/menus/action-strip-paragraph" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="action-strip-paragraph-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+<div class="divider--half"></div>
+
+### Usage
+In order to initialize and position correctly the Action Strip it needs to be inside relative container:
+
+```html
+<div style="position:relative;width:100px;height:100px;">
+    <igx-action-strip>
+        <button igxButton (click)="makeTextBold()">
+            <igx-icon>format_bold</igx-icon>
+        </button>
+    </igx-action-strip>
+<div>
+```
+
+By default the Action Strip will be visible, but this can be controlled by the [`hidden`]({environment:angularApiUrl}/classes/igxactionstripcomponent.html#hidden) @Input property.
+
+#### Menu look and feel
```

Comment:
```suggestion
#### Look and Feel
```

Reply:
This section is about the menu look and feel of the action strip. Any other idea for a header?

Reply:
I think you can dismiss this suggestion